pkg/datasource/builder.go

```diff
@@ -83,8 +97,19 @@ func (fb *FrameBuilder) Init(metadata *influxdb2.FluxTableMetadata) error {
 }
 
 func (fb *FrameBuilder) Append(record *influxdb2.FluxRecord) {
-	for i, key := range fb.names {
+	for _, key := range fb.names {
 		val := record.ValueByKey(key)
-		fb.frame.Fields[i].Append(val)
+		switch {
+		case isTag(key):
+			fb.fields["_value"].Labels[key] = val.(string)
```

Comment:
IIUC, this will overwrite the existing value at every row... but that is OK because it is the same?